### PR TITLE
Add PepQuery

### PR DESCRIPTION
**Pull request by jraysajulga**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/pepquery/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+source:
+  url: https://github.com/jraysajulga/PepQuery/archive/v{{ version }}.zip
```
Comment (bgruening) [inline recipes/pepquery/meta.yaml]: @jraysajulga is that a fork?
There is also a new version available at http://pepquery.org/data/PepQuery_v1.2.0.tar.gz

File: recipes/pepquery/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+requirements:
+  host:
+    - openjdk
+  run:
+    - openjdk
+
+test:
+  commands:
+    - pepquery -printPTM >/dev/null; rm -rf logs/
```
Comment (bgruening) [inline recipes/pepquery/meta.yaml]: you do not need to remove the logs in a test ... tests will not end up in a package

File: recipes/pepquery/meta.yaml (hunk 1 above)
Comment (jraysajulga) [reply recipes/pepquery/meta.yaml]: @bgruening Ah. Yes. I was meaning to update...

File: recipes/pepquery/meta.yaml (hunk 2 above)
Comment (jraysajulga) [reply recipes/pepquery/meta.yaml]: I had permission issues with the log files generated through the test command through CircleCI building and testing:

`File "/home/circleci/project/miniconda/lib/python3.6/shutil.py", line 436, in _rmtree_safe_fd
    os.unlink(name, dir_fd=topfd)
PermissionError: [Errno 13] Permission denied: 'log4j.properties'
Exited with code 1`

Comment (jraysajulga) [reply recipes/pepquery/meta.yaml]: Builds and tests fine on my local instance though...

Comment (bgruening) [reply recipes/pepquery/meta.yaml]: ok, wired, can you maybe add a comment about this to the recipe

Comment (jraysajulga) [reply recipes/pepquery/meta.yaml]: Sure. Not sure if I added a comment with best practices, so feel free to check.

**Comment by jraysajulga** [comment]
@bgruening Thanks for helping me out with this!

**Comment by bgruening** [comment]
You are very welcome!

**Pull request merged**